test/bdd/features/permissioned_data.feature

```diff
@@ -26,3 +26,26 @@ Feature: Permissioned data features
   When DV exports the last imported dataset as OT-JSON
   And DV waits for export to finish
   Then the last import should be the same on DC and DV nodes
+
+
+  @first
+  Scenario: Basic purchase scenario where buyer should receive private data while seller should take payment
+    Given the replication difficulty is 1
+    And I setup 4 nodes
+    And I start the nodes
+    And I use 1st node as DC
+    And DC imports "importers/use_cases/marketplace/permissioned_data_simple_sample.json" as OT-JSON
+    And DC waits for import to finish
+    Given DC initiates the replication for last imported dataset
+    And I wait for replications to finish
+    Given I use 2nd node as DV
+    And DV gets the list of available datasets for trading
+    And DV gets the price for the last imported dataset
+    And DV initiates purchase for the last imported dataset and waits for confirmation
```

Comment:
This should be two different steps

Reply:
We can't catch log event if we split this in two different steps